src/main/features/claim/views/defendant-represented.njk

```diff
@@ -32,7 +32,7 @@
         </fieldset>
 
         <div class="form-group panel panel-border-narrow js-hidden" id="company-name">
-          {{ textInput(t('Defendant representative organisation name'), 'organisationName', form) }}
+          {{ textInput(t('Representative's organisation name'), 'organisationName', form) }}
```

Comment:
This will require change in integration tests as well. Please make sure that `defendant-represented.js` inside `legal-integration-tests` is also updated accordingly.